app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] The application should ask confirmation of email. To do that you have to uncomment `confirmation` in device migration file. When `confirmable` is on, you will get the link in rails console no matter which email you enter. After, you can just copy the url to confirm and paste in browser.